libs/hook-dapp-lib/src/utils.ts

```diff
@@ -1,6 +1,18 @@
 import * as hookDappsRegistry from './hookDappsRegistry.json'
 import { CowHook, HookDappBase } from './types'
 
+const hookDapps = Object.keys(hookDappsRegistry).reduce((acc, id) => {
+  const dapp = (hookDappsRegistry as Record<string, Omit<HookDappBase, 'id'>>)[id]
+
+  acc.push({ id, ...dapp })
+  return acc
+}, [] as HookDappBase[])
+
+// permit() function selector
+const PERMIT_SELECTOR = '0xd505accf'
+// TODO: remove it after 01.01.2025
+const PERMIT_DAPP_ID = '1db4bacb661a90fb6b475fd5b585acba9745bc373573c65ecc3e8f5bfd5dee1f'
+
```

Comment:
Are you sure it can be removed?
Wouldn't this be needed to identify the hooks on the Explorer for example?
We could keep a list of ids that match the permit selector.